test/result/transaction/autocommit.result

```diff
@@ -0,0 +1,77 @@
+set autocommit=1;
+create table t1(a int);
+start transaction;
+insert into t1 values(1);
+start transaction;
+select * from t1;
+a
+1
+insert into t1 values(2);
+select * from t1;
+a
+1
+2
+rollback;
+select * from t1;
+a
+1
+start transaction;
+create table t2(a int);
+insert into t2(1);
+rollback;
+select * from t2;
+table "t2" does not exist
+start transaction;
+use autocommit;
+start transactions;
+drop table t1;
+Only CREATE of DDL is supported in transactions
+commit;
+start transactions;
+set autocommit=1;
+Parameters modification is unsupported in transactions
+commit;
+start transactions;
+show tables;
+tables_in_autocommit
+t1
+explain select * from t1;
+QUERY PLAN
+Project(cost=0.00..0.00 card=1.00 ndv=0.00 rowsize=0
+  ->  Table Scan on explain.t1(cost=0.00..0.00 card=1.00 ndv=0.00 rowsize=0
+commit;
+set autocommit=0;
+drop table if exists t1;
+create table t1(a int);
+insert into t1 values(1);
+rollback;
+select * from t1;
+a
+insert into t1 values(1);
+insert into t1 values(2);
+set autocommit=1;
+Uncommmited transaction exists. Please commit or rollback first.
+commit;
+set autocommit=1;
+set autocommit=0;
+select * from t1;
+a
+1
+2
+insert into t1 values(3);
+create table t2 (a int);
+rollback;
+select * from t2;
+a
+insert into t1 values(3);
+a
+1
+2
+3
+explain select * from t1;
+QUERY PLAN
+Project(cost=0.00..0.00 card=3.00 ndv=0.00 rowsize=0
+  ->  Table Scan on explain.t1(cost=0.00..0.00 card=3.00 ndv=0.00 rowsize=0
```

Comment:
Query plan like this probably will not be stable in the future.   rowsize, ndv, may get real values.   This is actually probably a good test, because data is so simple that we expect deterministic stats.   As long as we know the test stableness issue.